tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -511,7 +498,8 @@ public function test_get_report( $access_token ) {
 		);
 
 		$http_client = $this->create_fake_http_client( $property_id );
-		$this->analytics->get_client()->setHttpClient( $http_client );
+		$this->authentication->get_oauth_client()->get_client()->setHttpClient( $http_client );
+		$this->analytics->get_owner_oauth_client()->get_client()->setHttpClient( $http_client );
```

Comment:
This looks like it was missing initially but may have been working by coincidence. This will need to change going forward as using an alternate HTTP client won't work for this with Guzzle 6.